src/app/modules/item/pages/item-current-situation/item-current-situation.component.html

```diff
@@ -1,24 +1,52 @@
 <ng-container *ngIf="itemData">
   <div class="selection-center" *ngIf="!hideSelection">
-    <alg-selection (change)="onViewChanged($event)" [items]="viewItems" [selected]="viewSelectedIndex"></alg-selection>
+    <nav class="tab-rounded">
+      <a class="tab-rounded-item"
+         [ngClass]="{'active': historyTab.isActive, 'prev': chapterTab?.isActive}"
+         [routerLink]="'./progress/history'"
+         routerLinkActive #historyTab="routerLinkActive"
+      >
+        {{ logItemCaption }}
+      </a>
+
```

Comment:
There is an unwanted line between the tabs
![Screenshot 2021-05-26 at 14 27 32](https://user-images.githubusercontent.com/1053150/119659554-8be8e080-be2e-11eb-9745-4e43810d2a72.png)

Reply:
Fixed